packages/loctool/lib/CustomProject.js

```diff
@@ -241,17 +241,9 @@ CustomProject.prototype.defineFileTypes = function() {
  * extraction is done
  */
 CustomProject.prototype.extract = function(cb) {
-    this.parent.prototype.extract.call(this, function() {
-        this.db.getBy({
-            project: this.options.id,
-        }, function(err, resources) {
-            logger.trace("Getting all resources. Length: " + resources.length);
-            // logger.trace("Getting all resources. tu length: " + this.db.ts.resources.length);
-            this.translations.addAll(resources);
-
-            cb();
-        }.bind(this));
-    }.bind(this));
+    this.parent.prototype.extract.call(this, () => {
+        cb();
+    });
```

Comment:
```suggestion
```

If you just remove the method entirely, then any call to CustomProject.extract will go to the parent's implementation of extract, which is what we want in this case.